resources/ory/profile-evaluation.yaml

```diff
@@ -5,24 +5,32 @@ global:
       persistence:
         enabled: false
         postgresql:
-          enabled: false
+          enabled: true
         gcloud:
           enabled: false
 hydra:
   replicaCount: 1
   hydra:
-    autoMigrate: false
+    autoMigrate: true
   deployment:
     autoscaling:
       enabled: false
     resources:
       requests:
-        cpu: "10m"
+        cpu: 10m
 oathkeeper:
   replicaCount: 1
   deployment:
     resources:
       requests:
-        cpu: "10m"
+        cpu: 10m
     autoscaling:
       enabled: false
+postgresql:
```

Comment:
The postgresql config is also part of the ory azure-lite override and can be removed.